models/aeif_cond_alpha.nestml

```diff
@@ -86,11 +86,11 @@ neuron aeif_cond_alpha_neuron:
 
     # Impulse to add to DG_EXC on spike arrival to evoke unit-amplitude
     # conductance excursion.
-    PSConInit_E nS/ms = 1.0 nS * e / tau_syn_ex
+    PSConInit_E nS/ms = nS * e / tau_syn_ex
```

Comment:
I think the unit should always go with the quantity it applies to. In this case, `tau_syn_ex` carries its own unit (`ms`). The question is if one now should consider `e` as a dimensionless constant, in which case the right-hand side should be `1 nS * e / tau_syn_ex`, or assign `e` units, i.e., `e nS / tau_syn_ex`. I think I like the first version better; one might want to look again at the derivation of this expression to see where the `nS` actually comes from.